<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 163, 72, 255), stop:1 rgba(106, 0, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>video.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: oblique 16pt &quot;URW Gothic&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>Youtube Name</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>630</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: italic 18pt &quot;Ubuntu&quot;;
background-color: rgb(98, 160, 234);
color:white;</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
